wallboxplatform.js

```diff
@@ -261,6 +273,9 @@ class wallboxPlatform {
 			if(this.showControls==5 || this.showControls==4){outletService=lockAccessory.getServiceById(Service.Outlet, chargerID)}
 			if(this.showControls==3 || this.showControls==4){controlService=lockAccessory.getServiceById(Service.Thermostat, chargerID)}
 			if(this.showControls==1 || this.showControls==4){switchService=lockAccessory.getServiceById(Service.Switch, chargerID)}
+      let temperatureService=lockAccessory.getServiceById(Service.TemperatureSensor, chargerID)
+      let batteryPercent=this.calcBattery(batteryService,added_kWh,chargingTime)
+      let tempPercentage=(batteryPercent-32+.01)*5/9
```

Comment:
What happens if Celsius?

Reply:
Did that get left out of my pr commit, thought I had a condition on here. Let me double check.

Reply:
Looks like it was, added back in.